### DISCUSSION TITLE
Can Kiity support a afred like command window for quick switch between tabs/windows/shell scripts and so on

### DISCUSSION THREAD

**Discussion by zuozuo** [kovidgoyal/kitty#8176] [Ideas]
I met kitty yesterday, and I loved it once I tried. It's my favorite now.

![image](https://github.com/user-attachments/assets/e63570dd-f6ca-433e-9747-c61d53ac192e)
Can kitty support feature like this in neovim ?

In iterm2 i can select different profiles like below:
![image](https://github.com/user-attachments/assets/e600e142-7b39-4f3f-bfea-3bdb97ba8cbc)

Kitty has the awesome feature to start new tab and new window by remote control and startup session, i know i can create a lot of shell script to achieve the same feature like Iterm2 above.
However the shell script can only be executed in the terminal, when i am working in current tab with neovim or any other command line apps, i need to open a new tab to run my script. It's not so convenient.

**Comment by kovidgoyal** [comment]
you want to switch tabs by seelcting from a list use the select_tab action see https://sw.kovidgoyal.net/kitty/actions/
